app/views/layouts/application.html.erb

```diff
@@ -14,3 +14,7 @@
     <%= yield %>
   </body>
 </html>
+
+<% if flash[:alert].present? %>
+    <p class="my_alert"><%= flash[:alert] %></p>
+<% end %>
```

Comment:
In terms of user experience, you don't want to put the (important!) error messages all the way at the bottom because the rest of the page content will push it down and potentially hide it from the user. In term of code, you can't have `<p>` tags outside of `</body>` (in fact, _nothing_ but a doctype should go outside of the `<html>` tag), so this is invalid HTML. Where the flash message really belongs is above the `yield`.